server/uploadHandler/src/createNewJob.ts

```diff
@@ -5,6 +5,7 @@ import { getUploadedFilesInJob, getUploadURL } from "./service/S3Client"
 import { modelTiggerSqsEvent, sendMessage } from "./service/sqsClient"
 
 const seenomalySqsURL = process.env.SEENORMALY_URL as string;
+const owlEyeSqsURL = process.env.OWLEUE_URL as string;
 const videoExtension = ["mp4"];
 const imageExtension = ["jpg", "jpeg"];
```

Comment:
We also support .png, .jfif, .pjpeg, .pjp

Reply:
I will add png, but I have never seen the other three before and they aren't in the common [Mime-types ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)in HTTP requests.

Reply:
If you look here https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types and scroll down to the table, .jpg, .jpeg, .jfif, .pjpeg, .pjp all come under the image/jpeg MIME type